### ISSUE TITLE
ng2-bootstrap standalone npm install fails

### ISSUE THREAD

**Issue by sky-coding** [valor-software/ngx-bootstrap#825]
I freshly cloned development branch and ran `npm install`, and then `prepublish` triggers `flow.clean` and it fails:

```
ng2-bootstrap@1.0.24 flow.clean C:\git\ng2-bootstrap
> del bundles coverage demo-build typings "components/**/*.+(js|d.ts|js.map)" dist "ng2-bootstrap.+(js|d.ts|js.map)"

Could Not Find C:\git\ng2-bootstrap\bundles
The filename, directory name, or volume label syntax is incorrect.
```

Manually working around this step, I make it to `flow.compile:system` which fails:

```
ng2-bootstrap@1.0.24 flow.compile:system C:\git\ng2-bootstrap
> ./.config/bundle-system.js

'.' is not recognized as an internal or external command,
operable program or batch file.
```

Changing the command to `node ./.config/bundle-system.js`  fixes this error, but then the file fails:

```
Unhandled rejection Error on fetch for C:\git\ng2-bootstrap\node_modules\rxjs/Observable at file:///C:/git/C:\git\ng2-bootstrap\node_modules\rxjs/Observable.js
        Loading ng2-bootstrap/components/typeahead/typeahead.directive
        Loading ng2-bootstrap/components/typeahead
        Loading ng2-bootstrap/ng2-bootstrap
        Error: ENOENT: no such file or directory, open 'C:\git\C:\git\ng2-bootstrap\node_modules\rxjs\Observable.js'
    at Error (native)
```

What am I missing here? Are Windows developers unable to build this project? It's hard to contribute when `npm install` fails so hard.

**Comment by adrianfaciu** [comment]
Have a look at [this issue](https://github.com/valor-software/ng2-bootstrap/issues/493) also. There are some issues with the scripts on windows. I was able to manually bypass/fix a few things in order to make it work. Not sure though what is causing your last error though.

**Comment by valorkin** [comment]
should be fixed with #953
